src/stylus/components/_text-fields.styl

```diff
@@ -159,11 +159,15 @@ rtl(v-text-field-rtl, "v-text-field")
       background-position: bottom
       background-size: 3px 1px
       background-repeat: repeat-x
```

Comment:
Anyone know what all this crap is for? Probably isn't needed any more if we're using borders now.

Reply:
@KaelWD I'm not sure either, I tried removing those lines in the codepen, but there was no visible effect so I just left it as is.

By the way, sorry to bother, but I've noticed that I accidentally added `height` rules. Will I be able to edit the PR if I click that "pen" icon under the "Files changed" tab on the top and just remove those lines there?

Reply:
It's for disabled styles